packages/ckeditor5-modern/example/index.tsx

```diff
@@ -3,7 +3,12 @@ import * as React from 'react';
 import * as ReactDOM from 'react-dom';
 import { ChakraProvider, extendTheme, Flex, Button } from '@chakra-ui/react';
 
-import { ModernEditor } from '../.';
+import { ModernEditor } from './dist';
+
+localStorage.setItem(
+  'access_token',
+  'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOnsidHlwZSI6IkJ1ZmZlciIsImRhdGEiOlswLDE3OSw5OCw4NCwxOCwxMjcsMTA4LDI0OSw1MCwyNDcsOSw0NiwxMDksMjA0LDIzNCw3NV19LCJpYXQiOjE2Mjk0MzY1ODR9.ru0xRz2dN9SEgobwC1YdjCUHKRfBJlvNao7xu3yNdVw'
+);
```

Comment:
@nberongoy add a comment above this line, just a note that this is only for testing